cpp/tests/reductions/segmented_reduction_tests.cpp

```diff
@@ -387,6 +387,47 @@ TEST_F(SegmentedReductionTestUntyped, ReduceEmptyColumn)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*res, expect);
 }
 
+TEST_F(SegmentedReductionTestUntyped, EmptyInputWithOffsets)
+{
+  // values:    {}
+  // offsets:   {0, 0}
```

Comment:
Looking through this made me wonder about what happens with:
```
  // values:    {}
  // offsets:   {0, 0, 0, 0, 0, 0}
```
i.e., specifying multiple empty segments. Looking through the code this _should_ still work, but maybe let's modify this test to ensure it does work?

Reply:
Is that different than the string test, which includes multiple segments whose offsets are all zero? I thought this case might already be covered. https://github.com/rapidsai/cudf/blob/405a7d3b83fd90287dcf111c9abceb544599fd6f/cpp/tests/reductions/segmented_reduction_tests.cpp#L761